### ISSUE TITLE
There was an error download image container from azure container repository

### ISSUE THREAD

**Issue by leomindez** [Azure/service-fabric-issues#1108]
I am using service fabric to deploy some container images. 
I created the image containers and I uploaded them to Azure Containers Repository. 
I am creating a service fabric container application and I am adding the azure container url. 

When I do the deploy locally, I am getting these errors: 

```
Error event: SourceId='System.FM', Property='State'.
Partition is below target replica or instance count.
fabric:/Merchantapi/GuestMerchantType 1 1 9abf0ea1-9b90-4320-976c-e1c4ae8acc48
  IB _Node_4 Up 131733073645873100
  (Showing 1 out of 1 replicas. Total available replicas: 0)
```
``` 
Error event: SourceId='System.Hosting', Property='Download:1.0:1.0'. There was an error during download.Container xxxxxxx.azurecr.io/authorization.api:v2
```

I downloaded the images by using docker pull command and run them, the images work very well.

**Comment by MikkelHegn** [comment]
"Partition is below target replica or instance count." means that you do not have the container running. You asked for one replica, but there are zero running.

"Failed to download container image". Did you specify the credentials to ACR in the manifest file that you used to deploy the container with? https://docs.microsoft.com/en-us/azure/service-fabric/service-fabric-get-started-containers#configure-container-registry-authentication or

**Comment by leomindez** [comment]
Yes I did, I added the user and pass in application file. I have one question, I read that Windows Containers don't work on Windows 10, is it correct? 
Thanks for helping me
@MikkelHegn

**Comment by MikkelHegn** [comment]
Containers are supported on Windows 10, but there are some known limitations documented here: https://docs.microsoft.com/en-us/azure/service-fabric/service-fabric-how-to-debug-windows-containers

Can you share the docker host logs from Event Viewer. The error bobbles up from Docker and it would be interesting to see what's going on.
